test/cli/remote-table-upload/table-create-file-without-headers/in/data.csv

```diff
@@ -0,0 +1,4 @@
+"f030ed64cbc8babbe50901a26675a2ee","CSK Auto","US West","Active","2015-01-23"
```

Comment:
Add header

Reply:
Why? I think there is no reason to add headers because this is a test that the csv file does not have column names.